frame/scored-pool/src/lib.rs

```diff
@@ -407,16 +407,20 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
 		new_members.sort();
 
 		let old_members = <Members<T, I>>::get();
+		// TODO: What to do in case of error?
```

Comment:
regarding changes to this pallet. If you look more closely at the pallet code, you will see there is a storage item `MemberCount` which determines the number of members selected from the pool.

If I understand the end goals here correctly, you will make it so there is an upper limit to the number of members that can exist in a collective. So you should probably bring that information about the collective upper limit into this pallet, and choose the smaller of the two numbers when creating a membership group... or just make sure that if you send N members from this pallet to membership, that the top M members are selected or something like this.

Reply:
We will have to expose `MaxMembers` through `ChangeMembers`.
And while looking into this I also found that we need to adjust `InitializeMembers`.
This is snowballing :mountain_snow: